pages/exercises/[lessonSlug].tsx

```diff
@@ -39,7 +39,11 @@ const Exercises: React.FC<QueryDataProps<GetExercisesQuery>> = ({
       ? [{ text: 'lessons', url: currentLesson.docUrl }]
       : []),
     { text: 'challenges', url: `/curriculum/${currentLesson.slug}` },
-    { text: 'exercises', url: `/exercises/${currentLesson.slug}` }
+    { text: 'exercises', url: `/exercises/${currentLesson.slug}` },
+    {
+      text: 'mentor exercises',
+      url: `/curriculum/${currentLesson.slug}/mentor/`
```

Comment:
We should use a single `NavBar` across the curriculum pages. It'll be done in a different PR.